include/realizations/catchment/Bmi_Multi_Formulation.hpp

```diff
@@ -579,7 +579,7 @@ namespace realization {
             try {
                 auto const& nested_module = data_provider_iter->second;
                 long nested_module_time = nested_module->get_data_start_time() + ( this->get_model_current_time() - this->get_model_start_time() );
-                auto selector = CatchmentAggrDataSelector(this->get_catchment_id(),var_name,nested_module_time,this->record_duration(),"1");
+		auto selector = CatchmentAggrDataSelector(this->get_catchment_id(),var_name,nested_module_time,this->record_duration(),"");
```

Comment:
Whitespace got messed up here. This was fixed in #40, but this branch apparently was a `cherry-pick` rather than a `rebase`?